package.json

```diff
@@ -4,24 +4,35 @@
   "description": "A simple style guide generator for Grunt.js",
   "main": "Gruntfile.js",
   "dependencies": {
-    "nunjucks": "3.1.6",
+    "nunjucks": "3.2.0",
     "underscore": "1.8.3",
-    "yaml-front-matter": "3.2.3"
+    "yaml-front-matter": "4.1.0"
   },
   "devDependencies": {
-    "grunt": "1.0.3",
+    "@commitlint/cli": "8.3.4",
+    "@silvermine/eslint-config": "2.3.0",
+    "check-node-version": "4.0.2",
+    "grunt": "1.0.4",
     "grunt-browserify": "5.3.0",
-    "grunt-contrib-jshint": "2.0.0",
-    "grunt-sass": "3.0.2"
+    "grunt-contrib-jshint": "2.1.0",
+    "grunt-sass": "3.1.0",
+    "node-sass": "4.13.0"
   },
   "peerDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-browserify": "^3.8.0",
-    "grunt-sass": "^2.0.0"
+    "grunt": "~1.0.0",
+    "grunt-browserify": "~5.3.0",
+    "grunt-sass": "~3.0.0",
+    "node-sass": "~4.12.0"
```

Comment:
It seems we've added a new peer dependency on `node-sass` here. Is that really necessary? Isn't `grunt-sass` agnostic of whether you use `node-sass` or `dart-sass`? It seems like specifying SCSS implementation here just arbitrarily limits what can be used. There may be a reason that I'm missing though, so let me know.

Reply:
This also makes me wonder... doesn't `grunt-sass` 3+ require that you specify an `implementation` key in `options` that points to the implementation of SCSS that you want to use (node-sass vs dart-sass)? If so, does this plugin still work after this upgrade? I don't see any changes to the task file to pass in the `implementation`. Is it getting it some other way?

If we are required to specify the `implementation` in this task, maybe we can somehow get it from the configuration for `grunt-styleguide` that users of this plugin specify so that we can avoid creating a hard dependency on `node-sass`.

Reply:
Please see current installation and usage notes for [grunt-sass](https://github.com/sindresorhus/grunt-sass#install). The package no longer declares direct dependencies on either sass implementation, so it's up to the consumer to install their desired version.

You're right about the implementation option needing to be specified. I neglected to check the tasks directory for sass-related tasks and assumed it was up to the consumer to roll their own in their project's Gruntfile. I'll fix that asap.